server/views/pages/categoriserDone.html

```diff
@@ -15,7 +15,7 @@
   <h1 class="govuk-heading-l">Prisoner Categorisation</h1>
 </div>
 
-<div class="govuk-tabs" data-module="govuk-tabs">
+<div class="govuk-tabs" data-module="govuk-tabs-disabled">
```

Comment:
these are needed to fix a previously unnoticed console error caused by govuk update